lib/cc/engine/analyzers/violation.rb

```diff
@@ -32,12 +39,24 @@ def report_name
           "#{current_sexp.file}-#{current_sexp.line}"
         end
 
+        def mass # ABH Reverse engineer the actual mass from flay score.
```

Comment:
This feels like the big change here, and it's applicable across the board. If this can be cleanly done in it's own PR, I'd say go for that: it not, I would suggest editing the PR title/description to mention this in more detail, since those things will end up in commits & be the likely way people will search for the PR for historical context in the future. Your call, though.